dags/als-kp-dataset-dag.py

```diff
@@ -5,10 +5,11 @@
 
 1. Fetch data from the C-Path API using an authentication token stored in Airflow Variables
 2. Transform the raw data using a JSONata mapping expression and validate against a JSON Schema
-3. Create or update Synapse Datasets for each item in the transformed data
-4. Create or update a Dataset Collection containing all the datasets
-5. Create or update annotations for each dataset in the collection
-6. Create a new snapshot of the collection if any changes were detected
+3. Find duplicates in the new C-Path data and send a message to AMP-ALS slack channel if duplicates are found. 
+4. Retrieve datasets that need to be ignored after human validation from a JSON file
```

Comment:
You may want to add the synapse id of the file here, and state that it is the responsibility of the data managers in AMP-ALS to update the JSON file

Reply:
Sure. Let me add that.